### PR TITLE
Reduce scrollbar opacity to show minimap colours

### LINKED ISSUE #5: Issue respective to git
Thanks for the awesome theme, found some issue after installing for the first time

https://user-images.githubusercontent.com/22526247/208661352-ea5919fa-0710-4609-989f-2c6dee9550e6.mov

scrollbar is hiding the difference indicator

https://user-images.githubusercontent.com/22526247/208661546-05025b3c-7587-47da-8cb8-ae9b24e479df.mov

Selected text is not visible

### PR DESCRIPTION
**Pull request by VoxelMC**
### Description

Reduces the opacity of the scrollbar so that colours on the minimap - such as Git additions and deletions and errors - are visible underneath it for all states (hover, idle, and active).

https://github.com/withastro/houston-vscode/assets/20650404/e9c9341f-454c-4fd6-9490-61e3ef521167

### Related issues & labels

This PR does **not yet** close #5, however it addresses one of the issues mentioned.

### CODE REVIEW

**Comment by VoxelMC** [comment]
Will write a changeset later today.

File: themes/houston.json [hunk 1]
```diff
@@ -1960,9 +1960,9 @@
     "pickerGroup.foreground": "#eef0f9",
     "progressBar.background": "#4bf3c8",
     "scrollbar.shadow": "#000000",
-    "scrollbarSlider.activeBackground": "#54b9ff",
-    "scrollbarSlider.background": "#343841",
-    "scrollbarSlider.hoverBackground": "#545864",
+    "scrollbarSlider.activeBackground": "#54b9ff99",
+    "scrollbarSlider.background": "#343841DD",
+    "scrollbarSlider.hoverBackground": "#54586499",
```
Comment (delucis) [inline themes/houston.json]: What do you think of these tweaks?

```suggestion
    "scrollbarSlider.activeBackground": "#54b9ff66",
    "scrollbarSlider.background": "#54586466",
    "scrollbarSlider.hoverBackground": "#545864B3",
```

Looks like VS Code’s [default opacities](https://github.com/microsoft/vscode/blob/a9c390b23628adedadb0116de2b258524abcb838/src/vs/platform/theme/common/colorRegistry.ts#L278-L280) here are 40% (`66` in hexadecimal) for the background shades, and 70% (`B3` in hex) for the hover shade.

By reducing the opacity but brightening the colours we can let more of the background minimap details through while still keeping a similar contrast between the scrollbar elements.

**Review by delucis — COMMENTED** [review]
Thanks for this improvement @VoxelMC! Left one suggestion.

File: themes/houston.json (hunk 1 above)
Comment (VoxelMC) [reply themes/houston.json]: Change makes sense, I will have a look at it when I get home, then commit it! Thank you for looking at it :)

**Review by delucis — APPROVED** [review]
Thanks again @VoxelMC! Added the changeset and will get this released now.

**Pull request merged**